lib/models/operation.js

```diff
@@ -31,8 +31,9 @@ class Operation extends OperationTraitable {
    * @returns {Message}
    */
   message(index) {
-    if (typeof index !== 'number' || !this._json.message) return null;
+    if (!this._json.message) return null;
     if (!this._json.message.oneOf) return new Message(this._json.message);
+    if (typeof index !== 'number') return null;
```

Comment:
I don't get these changes. What's different?

Reply:
`if (typeof index !== 'number') return null;` is a security requirement to always make sure provided index is a number

but when it was like this `if (typeof index !== 'number' || !this._json.message) return null;`, even if you did not provide index (this was possible with `message` function) you were always getting `null` becouse `undefined` is also not a `number`

so now I check for the number right before the part of the message where index is used and most important is that this part `    if (!this._json.message.oneOf) return new Message(this._json.message);` now works as before

Reply:
Oh, I see! Thanks for the explanation 🙏